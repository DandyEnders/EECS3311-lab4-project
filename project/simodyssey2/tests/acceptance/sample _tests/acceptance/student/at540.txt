move(NW)
liftoff
move(SE)
move(N)
liftoff
move(N)
move(W)
move(E)
liftoff
move(E)
move(SW)
move(N)
move(E)
move(S)
move(SW)
play
liftoff
move(N)
move(E)
pass
wormhole
move(SW)
move(E)
move(NE)
wormhole
move(S)
wormhole
move(SE)
move(SW)
move(S)
abort
move(S)
wormhole
move(SE)
move(NE)
abort
pass
move(W)
move(E)
move(NW)
move(NW)
move(NW)
move(NE)
pass
move(W)
play
move(SE)
play
move(N)
move(E)
pass
move(SE)
move(E)
move(SW)
move(E)
move(E)
move(SE)
move(NW)
wormhole
abort
move(NE)
pass
move(N)
move(E)
move(W)
move(E)
move(NE)
move(S)
move(W)
move(E)
wormhole
land
move(SW)
