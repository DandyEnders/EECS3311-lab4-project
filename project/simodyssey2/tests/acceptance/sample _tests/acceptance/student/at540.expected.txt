  state:0.0, ok
  Welcome! Try test(3,5,7,15,30)
->move(NW)
  state:0.1, error
  Negative on that request:no mission in progress.
->liftoff
  state:0.2, error
  Negative on that request:no mission in progress.
->move(SE)
  state:0.3, error
  Negative on that request:no mission in progress.
->move(N)
  state:0.4, error
  Negative on that request:no mission in progress.
->liftoff
  state:0.5, error
  Negative on that request:no mission in progress.
->move(N)
  state:0.6, error
  Negative on that request:no mission in progress.
->move(W)
  state:0.7, error
  Negative on that request:no mission in progress.
->move(E)
  state:0.8, error
  Negative on that request:no mission in progress.
->liftoff
  state:0.9, error
  Negative on that request:no mission in progress.
->move(E)
  state:0.10, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.11, error
  Negative on that request:no mission in progress.
->move(N)
  state:0.12, error
  Negative on that request:no mission in progress.
->move(E)
  state:0.13, error
  Negative on that request:no mission in progress.
->move(S)
  state:0.14, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.15, error
  Negative on that request:no mission in progress.
->play
  state:1.0, mode:play, ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   P---   ----   PP--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    MY--   ----   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   M*--   P---   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    BW--   ----   *---   ----   PW--   
->liftoff
  state:1.1, mode:play, error
  Negative on that request:you are not on a planet at Sector:1:1
->move(N)
  state:2.0, mode:play, ok
  Movement:
    [0,E]:[1,1,1]->[5,1,3]
    [3,P]:[1,5,2]->[5,4,1]
    [8,P]:[4,3,1]->[3,2,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *---   P---   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    MY--   P---   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   M*--   ----   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    BWE-   ----   *---   P---   PW--   
->move(E)
  state:3.0, mode:play, ok
  Movement:
    [0,E]:[5,1,3]->[5,2,1]
    [5,M]:[3,1,1]->[3,5,1]
    [8,P]:[3,2,1]->[2,1,1]
    [10,B]:[5,1,1]->[5,2,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *---   P---   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    P---   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   ----   O---   PW--   M---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   M*--   ----   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   EB--   *---   P---   PW--   
->pass
  state:4.0, mode:play, ok
  Movement:
    [1,P]:[1,3,1]->[2,4,1]
    [2,P]:[1,5,1]->[2,5,2]
    [4,P]:[2,5,1]->[2,4,2]
    [6,P]:[3,4,1]->[2,3,2]
    [7,M]:[4,2,1]->[5,1,1]
    [10,B]:[5,2,2]->[5,1,3]
      reproduced [12,B] at [5,1,4]
      destroyed [7,M] at [5,1,1]
    [11,P]:[5,5,1]->[4,4,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *---   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    P---   ----   WP--   PP--   -P--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   ----   O---   -W--   M---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   -*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -WBB   E---   *---   P---   -W--   
->wormhole
  state:4.1, mode:play, error
  Explorer couldn't find wormhole at Sector:5:2
->move(SW)
  state:5.0, mode:play, ok
  Explorer got lost in space - out of fuel at Sector:1:1
  The game has ended. You can start a new game.
  Movement:
    [0,E]:[5,2,1]->[1,1,1]
    [2,P]:[2,5,2]->[1,5,1]
    [3,P]:[5,4,1]->[1,4,1]
    [6,P]:[2,3,2]->[3,2,1]
    [8,P]:[2,1,1]->[1,5,2]
    [12,B]:[5,1,4]->[5,3,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *---   ----   P---   PP--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   PP--   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   P---   O---   -W--   M---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   -*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -WB-   ----   *B--   ----   -W--   
->move(E)
  state:5.1, error
  Negative on that request:no mission in progress.
->move(NE)
  state:5.2, error
  Negative on that request:no mission in progress.
->wormhole
  state:5.3, error
  Negative on that request:no mission in progress.
->move(S)
  state:5.4, error
  Negative on that request:no mission in progress.
->wormhole
  state:5.5, error
  Negative on that request:no mission in progress.
->move(SE)
  state:5.6, error
  Negative on that request:no mission in progress.
->move(SW)
  state:5.7, error
  Negative on that request:no mission in progress.
->move(S)
  state:5.8, error
  Negative on that request:no mission in progress.
->abort
  state:5.9, error
  Negative on that request:no mission in progress.
->move(S)
  state:5.10, error
  Negative on that request:no mission in progress.
->wormhole
  state:5.11, error
  Negative on that request:no mission in progress.
->move(SE)
  state:5.12, error
  Negative on that request:no mission in progress.
->move(NE)
  state:5.13, error
  Negative on that request:no mission in progress.
->abort
  state:5.14, error
  Negative on that request:no mission in progress.
->pass
  state:5.15, error
  Negative on that request:no mission in progress.
->move(W)
  state:5.16, error
  Negative on that request:no mission in progress.
->move(E)
  state:5.17, error
  Negative on that request:no mission in progress.
->move(NW)
  state:5.18, error
  Negative on that request:no mission in progress.
->move(NW)
  state:5.19, error
  Negative on that request:no mission in progress.
->move(NW)
  state:5.20, error
  Negative on that request:no mission in progress.
->move(NE)
  state:5.21, error
  Negative on that request:no mission in progress.
->pass
  state:5.22, error
  Negative on that request:no mission in progress.
->move(W)
  state:5.23, error
  Negative on that request:no mission in progress.
->play
  state:6.0, mode:play, ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EPAW   BW--   ----   P---   PY--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    *---   ----   *---   B---   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   P---   O---   P---   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    P---   Y---   *---   P---   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    *---   A*--   P---   ----   *---   
->move(SE)
  state:7.0, mode:play, ok
  Explorer got destroyed by asteroid (id: 2) at Sector:2:2
  The game has ended. You can start a new game.
  Movement:
    [0,E]:[1,1,1]->[2,2,1]
    [2,A]:[1,1,3]->[2,2,2]
      destroyed [0,E] at [2,2,1]
    [7,P]:[3,2,1]->[2,3,2]
    [8,P]:[3,4,1]->[4,4,2]
    [10,P]:[4,4,1]->[4,3,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -P-W   BW--   ----   P---   PY--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    *---   -A--   *P--   B---   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   ----   O---   ----   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    P---   Y---   *P--   -P--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    *---   A*--   P---   ----   *---   
->play
  state:8.0, mode:play, ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EPB-   P---   P*--   ----   B---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    BW--   ----   Y---   B---   B---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   BB--   O---   JW--   B*--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    PY--   P*--   ----   BY--   B---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   P---   Y---   ----   BY--   
->move(N)
  state:9.0, mode:play, ok
  Movement:
    [0,E]:[1,1,1]->[5,1,1]
    [3,P]:[1,2,1]->[1,3,3]
    [5,B]:[1,5,1]->[1,4,1]
    [7,B]:[2,4,1]->[1,3,4]
    [10,B]:[3,2,2]->[4,3,1]
    [15,B]:[4,4,1]->[4,3,2]
    [18,B]:[5,5,1]->[4,4,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -PB-   ----   P*PB   B---   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    BW--   ----   Y---   ----   B---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   B---   O---   JW--   B*--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    PY--   P*--   BB--   BY--   B---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    E---   P---   Y---   ----   -Y--   
->move(E)
  state:10.0, mode:play, ok
  Movement:
    [0,E]:[5,1,1]->[5,2,2]
    [1,P]:[1,1,2]->[5,5,1]
    [2,B]:[1,1,3]->[1,2,1]
    [7,B]:[1,3,4]->[2,4,1]
      reproduced [19,B] at [2,4,2]
    [10,B]:[4,3,1]->[3,2,2]
      reproduced [20,B] at [3,2,3]
    [12,B]:[3,5,1]->[2,1,3]
    [15,B]:[4,3,2]->[4,2,3]
      reproduced [21,B] at [4,2,4]
    [16,B]:[4,5,1]->[4,4,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   B---   P*P-   B---   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    BWB-   ----   Y---   BB--   B---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   BBB-   O---   JW--   -*--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    PY--   P*BB   ----   BYB-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   PE--   Y---   ----   PY--   
->pass
  state:11.0, mode:play, ok
  Movement:
    [6,B]:[2,1,1]
    [8,B]:[2,5,1]->[3,4,3]
    [9,B]:[3,2,1]
    [11,J]:[3,4,1]->[2,5,1]
    [17,P]:[5,2,1]->[5,1,1]
    [18,B]:[4,4,1]->[4,3,1]
      reproduced [22,B] at [4,3,2]
    [20,B]:[3,2,3]->[3,1,1]
    [21,B]:[4,2,4]->[3,1,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   B---   P*P-   B---   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    BWB-   ----   Y---   BB--   J---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    BB--   BB--   O---   -WB-   -*--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    PY--   P*B-   BB--   -YB-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    P---   -E--   Y---   ----   PY--   
->move(SE)
  state:12.0, mode:play, ok
  Movement:
    [0,E]:[5,2,2]->[1,3,4]
    [2,B]:[1,2,1]->[2,2,1]
      reproduced [23,B] at [2,2,2]
    [5,B]:[1,4,1]->[1,5,1]
      reproduced [24,B] at [1,5,2]
    [6,B]:[2,1,1]->[3,5,1]
      reproduced [25,B] at [3,5,3]
    [9,B]:[3,2,1]->[4,1,3]
      reproduced [26,B] at [4,1,4]
    [10,B]:[3,2,2]->[4,2,4]
    [12,B]:[2,1,3]->[5,2,1]
      reproduced [27,B] at [5,2,2]
    [15,B]:[4,2,3]->[5,2,3]
    [16,B]:[4,4,3]->[5,4,1]
      reproduced [28,B] at [5,4,2]
    [17,P]:[5,1,1]
    [20,B]:[3,1,1]->[4,2,3]
    [22,B]:[4,3,2]->[3,4,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   ----   P*PE   ----   BB--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    -W--   BB--   Y---   BB--   J---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -B--   ----   O---   BWB-   B*B-   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    PYBB   P*BB   B---   -Y--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    P---   BBB-   Y---   BB--   PY--   
->move(E)
  state:13.0, mode:play, ok
  Movement:
    [0,E]:[1,3,4]->[1,4,1]
    [6,B]:[3,5,1]->[4,5,1]
    [7,B]:[2,4,1]->[1,5,3]
    [11,J]:[2,5,1]->[1,4,2]
    [12,B]:[5,2,1]->[5,3,2]
    [17,P]:[5,1,1]->[5,5,3]
    [18,B]:[4,3,1]
    [19,B]:[2,4,2]->[3,5,1]
    [23,B]:[2,2,2]->[1,3,4]
    [24,B]:[1,5,2]->[1,1,1]
    [25,B]:[3,5,3]->[2,5,1]
    [28,B]:[5,4,2]->[4,3,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    B---   ----   P*PB   EJ--   B-B-   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    -W--   B---   Y---   ----   B---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -B--   ----   O---   BWB-   B*--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    PYBB   P*BB   BB--   -Y--   B---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   -BB-   YB--   B---   PYP-   
->move(SW)
  state:14.0, mode:play, ok
  Movement:
    [0,E]:[1,4,1]->[2,3,2]
    [8,B]:[3,4,3]->[2,4,1]
      reproduced [29,B] at [2,4,2]
    [9,B]:[4,1,3]
    [10,B]:[4,2,4]->[3,3,2]
    [15,B]:[5,2,3]
      reproduced [30,B] at [5,2,1]
    [16,B]:[5,4,1]->[5,3,3]
    [18,B]:[4,3,1]->[5,3,4]
    [20,B]:[4,2,3]->[5,1,1]
      reproduced [31,B] at [5,1,2]
    [21,B]:[3,1,2]->[3,2,1]
      reproduced [32,B] at [3,2,2]
    [22,B]:[3,4,1]->[5,4,1]
      reproduced [33,B] at [5,4,2]
    [26,B]:[4,1,4]->[5,5,4]
    [27,B]:[5,2,2]->[4,3,1]
    [28,B]:[4,3,2]->[3,4,1]
      reproduced [34,B] at [3,4,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    B---   ----   P*PB   -J--   B-B-   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    -W--   B---   YE--   BB--   B---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   BB--   O---   BWB-   B*--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    PYB-   P*--   B---   -Y--   B---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    BB--   B-B-   YBBB   BB--   PYPB   
->move(E)
  state:15.0, mode:play, ok
  Movement:
    [0,E]:[2,3,2]->[2,4,3]
    [2,B]:[2,2,1]->[1,1,2]
    [5,B]:[1,5,1]->[2,5,2]
    [7,B]:[1,5,3]->[1,1,2]
    [9,B]:[4,1,3]->[5,1,3]
      reproduced [35,B] at [5,1,4]
    [12,B]:[5,3,2]->[4,4,1]
      reproduced [36,B] at [4,4,3]
    [16,B]:[5,3,3]->[4,4,4]
    [19,B]:[3,5,1]->[2,1,1]
      reproduced [37,B] at [2,1,3]
    [22,B]:[5,4,1]->[1,5,1]
    [23,B]:[1,3,4]->[2,3,2]
      reproduced [38,B] at [2,3,3]
    [25,B]:[2,5,1]->[1,1,2]
      reproduced [39,B] at [1,1,3]
    [28,B]:[3,4,1]->[2,3,4]
    [29,B]:[2,4,2]->[1,5,2]
    [34,B]:[3,4,3]->[2,5,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    BBB-   ----   P*P-   -J--   BB--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    BWB-   ----   YBBB   B-E-   B---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   BB--   O---   -W--   -*--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    PY--   P*--   B---   BYBB   B---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    BBBB   B-B-   Y--B   -B--   PYPB   
->move(E)
  state:16.0, mode:play, ok
  Movement:
    [0,E]:[2,4,3]->[2,5,2]
    [6,B]:[4,5,1]->[3,4,1]
      reproduced [40,B] at [3,4,3]
    [11,J]:[1,4,2]->[5,3,2]
      reproduced [41,J] at [5,3,3]
    [18,B]:[5,3,4]->[1,3,4]
    [21,B]:[3,2,1]->[4,2,3]
    [24,B]:[1,1,1]->[2,2,1]
      reproduced [42,B] at [2,2,2]
    [31,B]:[5,1,2]->[1,5,3]
    [32,B]:[3,2,2]->[4,3,2]
    [33,B]:[5,4,2]->[1,5,4]
    [37,B]:[2,1,3]->[5,4,1]
    [39,B]:[1,1,3]->[2,1,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -B--   ----   P*PB   ----   BBBB   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    BWB-   BB--   YBBB   B---   BE--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   ----   O---   BWB-   -*--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    PY--   P*B-   BB--   BYBB   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    B-BB   B-B-   YJJ-   B---   PYPB   
->move(SE)
  state:17.0, mode:play, ok
  Explorer got lost in space - out of fuel at Sector:3:1
  The game has ended. You can start a new game.
  Movement:
    [0,E]:[2,5,2]->[3,1,1]
    [8,B]:[2,4,1]
    [9,B]:[5,1,3]
    [15,B]:[5,2,3]
    [18,B]:[1,3,4]->[1,4,1]
      reproduced [43,B] at [1,4,2]
    [19,B]:[2,1,1]->[3,1,1]
    [20,B]:[5,1,1]->[4,5,1]
    [22,B]:[1,5,1]
    [23,B]:[2,3,2]->[1,4,3]
    [26,B]:[5,5,4]->[5,4,2]
      reproduced [44,B] at [5,4,3]
    [27,B]:[4,3,1]->[5,3,4]
    [29,B]:[1,5,2]->[2,1,1]
      reproduced [45,B] at [2,1,4]
    [30,B]:[5,2,1]->[4,2,4]
    [31,B]:[1,5,3]->[2,5,2]
      reproduced [46,B] at [2,5,3]
    [33,B]:[1,5,4]->[5,4,4]
    [34,B]:[2,5,1]->[1,5,2]
      reproduced [47,B] at [1,5,3]
    [35,B]:[5,1,4]->[4,1,3]
    [36,B]:[4,4,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -B--   ----   P*P-   BBB-   BBB-   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    BWBB   BB--   Y-BB   B---   -BB-   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    B---   ----   O---   BWB-   -*--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    PYB-   P*BB   -B--   BYBB   B---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    --B-   --B-   YJJB   BBBB   PYP-   
->move(NW)
  state:17.1, error
  Negative on that request:no mission in progress.
->wormhole
  state:17.2, error
  Negative on that request:no mission in progress.
->abort
  state:17.3, error
  Negative on that request:no mission in progress.
->move(NE)
  state:17.4, error
  Negative on that request:no mission in progress.
->pass
  state:17.5, error
  Negative on that request:no mission in progress.
->move(N)
  state:17.6, error
  Negative on that request:no mission in progress.
->move(E)
  state:17.7, error
  Negative on that request:no mission in progress.
->move(W)
  state:17.8, error
  Negative on that request:no mission in progress.
->move(E)
  state:17.9, error
  Negative on that request:no mission in progress.
->move(NE)
  state:17.10, error
  Negative on that request:no mission in progress.
->move(S)
  state:17.11, error
  Negative on that request:no mission in progress.
->move(W)
  state:17.12, error
  Negative on that request:no mission in progress.
->move(E)
  state:17.13, error
  Negative on that request:no mission in progress.
->wormhole
  state:17.14, error
  Negative on that request:no mission in progress.
->land
  state:17.15, error
  Negative on that request:no mission in progress.
->move(SW)
  state:17.16, error
  Negative on that request:no mission in progress.
